test/axis-interval.html

```diff
@@ -338,6 +337,67 @@ <h1>[ Test time axis interval ]&nbsp;&nbsp;&nbsp; should not overlap when data z
         </script>
 
 
+<script>
+
+    require([
+        'echarts'
+    ], function (echarts) {
+
+        let dataIndex = 0;
+
+        let testArr = [
+            [true, false, true, true],
+            [true, false, true, false],
+            [false, true, true, false],
+            [false, true, false, true],
+            [false, false, true, true],
+            [true, true, false, false]
+        ];
+
+        function print(index){
+            return `intervalList = [${testArr[index].toString()}] ,currentIndex is ${index}`
+        }
+
+        var option = {
+            title:{
```

Comment:
Space is required before `:`, similar to other places.

Reply:
will pay attention next time

Reply:
@foolzhang Never mind. :) You can run `jshint` next time before you make a pull request.